### ISSUE TITLE
No Logging in fb.util

### ISSUE THREAD

**Issue by vermaeshan** [facebook/fb-caffe-exts#1]
Hey
On Running 
th torch2caffe/torch2caffe.lua --help
I get the following error:
"module 'fb.util.logging' not found:No LuaRocks module found for fb.util.logging"
I can't seem to find fb.util.logging implemented at:
https://github.com/facebook/fblualib/tree/master/fblualib/util
are you using a private repo?

**Comment by ajtulloch** [comment]
Ah, we haven't pushed the fb.util package recently - the logging util is
just a wrapper around glog for Lua. I'll get that updated. For now, I think
you can just comment out the logging (or convert them to print statements).

**Comment by vermaeshan** [comment]
Yeah, I got around that by installing "logging" which has similar capabilities and functions.

**Comment by likeucode** [comment]
Hi
when i run " th torch2caffe/torch2caffe.lua --help" in the CLI, i get an error:
 "module 'fb.util.logging' not found:No LuaRocks module found for
 fb.util.logging"
How can i figure it out?

**Comment by ajtulloch** [comment]
The steps in https://github.com/facebook/fb-caffe-exts/issues/3#issuecomment-185497935 will work - it's an incompatibility between FB's fblualib and the GitHub fblualib.

**Comment by likeucode** [comment]
@ajtulloch Thank you very much !
